test_enet.py

```diff
@@ -71,6 +71,24 @@ def test_connect(self):
         self.assertEquals(client_connected, True)
         self.assertEquals(server_connected, True)
 
+    def test_socketsend(self):
+
+        self.send_done = False
+        socketsend_msg = "\xff\xff\xff\xffgetstatus\x00"
+
+        def f(address, data):
+            if data == socketsend_msg:
+                self.send_done = True
+
+        self.server.intercept = f
+
+        while not self.send_done:
+
+            self.client.service(0)
```

Comment:
You likely want to follow some of the other examples here, maybe the broadcast test below? You should be checking if the event type is connect and then do the socket.send() call, then you can call the server.service() in subsequent loops waiting for the intercept call back to fire.  If you don't add that conditional in, you'll be calling socket.send() over and over again.